scripts/postprocessing/coverage_analysis/coverage_Analyzer.py

```diff
@@ -0,0 +1,624 @@
+
```

Comment:
Please name all the files in python style, coverage_analyzer.py, activity_db_generator, etc. No capitals.